scout/constants/indexes.py

```diff
@@ -17,6 +17,11 @@
             name="build_aliases",
             background=True,
         ),
+        IndexModel(
+            [("build", ASCENDING), ("hgnc_symbol", ASCENDING)],
+            name="build_symbol",
```

Comment:
```suggestion
            name="build_hgnc_symbol",
```

Reply:
I actually like your name better, but *cough* for reasons of compatibility with the current db perhaps it is easiest to use mine? 😬